src/react-components/ui-root.js

```diff
@@ -440,7 +440,7 @@ class UIRoot extends Component {
     if (!this.props.forcedVREntryType || !this.props.forcedVREntryType.endsWith("_now")) {
       this.goToEntryStep(ENTRY_STEPS.audio_setup);
     } else {
-      setTimeout(this.onAudioReadyButton, 3000); // Need to wait otherwise input doesn't work :/
+      this.onAudioReadyButton();
```

Comment:
Glad to see a mysterious `setTimeout` go away! Was there a race condition that got fixed?

Reply:
No it's not a fix really, it just changes the UX for the `vr_entry_type=XXX_now` so that it actually doens't just enter, it just makes it one-click to enter.